### PR TITLE
samples: mesh: nrf52: removed unwanted code

### PR DESCRIPTION
**Pull request by vikrant8052**
Removed unwanted code which is not allowing GET response
to get send by Server to client. After removing this, there
is improvement in GATT Bearer performance while testing
with Android nRFMesh App.

Signed-off-by: Vikrant More <vikrant8051@gmail.com>

### CODE REVIEW

**Comment by vikrant8051** [comment]
Please wait.

File: samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c [hunk 1]
```diff
@@ -101,6 +101,31 @@ static struct bt_mesh_elem elements[];
 
 /* message handlers (Start) */
 
+#define common()							       \
```
Comment (jhedberg) [inline samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c]: There are many things wrong with this:

 - It should be a helper function
 - It should have a descriptive name matching what it does
 - Even if it was a macro it should be all upper-case

**Review by jhedberg — CHANGES_REQUESTED** [review]

File: samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c (hunk 1 above)
Comment (vikrant8051) [reply samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c]: Removed that macro. So issue has solved !!

**Comment by vikrant8051** [comment]
@jhedberg 
After making changes as per 3rd commit, I could save #ResetCounter value on flash. Otherwise on every reboot it is not getting updated & remains Zero. That means changes made by https://github.com/zephyrproject-rtos/zephyr/pull/9521 have issue.

**Pull request closed without merging**